docs/docs/CMDB/FAQ.md

```diff
@@ -0,0 +1,209 @@
+# FAQ
+
+## Why not use existing models from NetBox?
+
+[NetBox](https://netbox.dev) is a great tool to manage your DCIM and IPAM.
+
+Our CMDB models directly use the DCIM and IPAM models, like `Device` and `IP prefixes`.
+
+NetBox also includes several models which could be considered as CMDB such as `VLAN`, `VRF` and `L2VPN`. We do not want to use these models to avoid confusion and design mismatches with our CMDB.
+
+## Why not implement directly OpenConfig models in the CMDB?
+
+OpenConfig models are designed from a device perspective.
+
+Our CMDB models are designed from a service/asset perspective. It enables us to apply common parameters without worrying of applying them at both ends.
+
+Example: we can set a maintenance status on a BGP session directly.
```

Comment:
Example: we can set a maintenance status on a BGP session directly. ->
Example: we can directly set a maintenance status on a BGP session.